docs/using-hive.md

```diff
@@ -188,21 +188,41 @@ oc get nodes
 
 Hive can optionally create delegated Route53 DNS zones for each cluster.
 
-To use this functionality you must first define the base domains you want to manage DNS for in your HiveConfig.
+To use this feature:
+
+  1. Manually create a Route53 hosted zone for your "root" domain (i.e. hive1.example.com in the example below) and ensure your DNS is operational.
+  1. Create a secret in the "hive" namespace with AWS credentials with permissions to manage the root hosted zone.
+     ```
+     apiVersion: v1
+     data:
+       aws_access_key_id: REDACTED
+       aws_secret_access_key: REDACTED
+     kind: Secret
+     metadata:
+       name: route53-aws-creds
+     type: Opaque
+     ```
+  1. Update your HiveConfig to enable externalDNS and set the list of managed domains:
+     ```
+     apiVersion: hive.openshift.io/v1alpha1
+     kind: HiveConfig
+     metadata:
+       name: hive
+     spec:
+       managedDomains:
+       - hive1.example.com
+       externalDNS:
+         aws:
+           credentials:
+             name: route53-aws-creds
+     ```
+
+You can now request clusters with clusterDeployment.spec.manageDNS=true and clusterDeployment.spec.baseDomain=mycluster.hive1.example.com. Hive will create a mycluster.hive1.example.com hosted zone and link it with your root hosted zone. Per-cluster DNS entries ("apps", "api", etc) will be created in the new mycluster.hive1.example.com hosted zone.
 
 ```
-apiVersion: hive.openshift.io/v1alpha1
-kind: HiveConfig
-metadata:
-  name: hive
-spec:
-  managedDomains:
-  - hive1.example.com
+$ bin/hiveutil create-cluster --base-domain=devcluster.hive.example.com devcluster --manage-dns
```

Comment:
Can you have this command match up with the names used in the preceding paragraph? I am having a hard time following along with how the base domain used by the installed relates to the domain of the resultant cluster.

Reply:
yes, this should read --base-domain=devcluster.hive1.example.com

Reply:
perhaps this would read better if it was --base-domain=devdomain.hive1.example.com. this means that hive is responsible for making devdomain.hive1.example.com NS entry in the hive1.example.com Route53 zone that points to the hive-created devdomain.hive1.example.com Route53 zone.